storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -2850,6 +2853,7 @@ public void launchServer() throws Exception {
                                             }
                                             doCleanup();
                                         } catch (Exception e) {
+                                            this.mkAssignmentsErrors.mark();
```

Comment:
should we have separate tracker for cleanup failures and `doCleanup()` ?

Reply:
My thought was any exception would be worth investigating in any case.  I would hope that these issues would be rare.  Let me know if you want it more narrowly applied just to mkAssignment.

Reply:
Yes, let's apply this narrowly to just `mkAssignments`